dist/Makefile.tmpl

```diff
@@ -134,6 +134,14 @@ include ctypes.depend
 
 all: libocamlevercrypt.cmxa
 
+lib/Lib_RandomBuffer_System_bindings.cmx:
```

Comment:
why not copy them into the `lib` and `lib_gen` directories respectively, and let the automatic dependencies kick in? wouldn't work? in any case please use `-c $<` and `-o $@` to minimize the risk of typos

Reply:
by that I mean that if they were in the right directories you wouldn't have to manually list the ocamlopt invocation